metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/tree_function.py

```diff
@@ -80,22 +80,14 @@ def token_values(tree: Tree, parameters: Dict[str, str] = {}) -> List[str]:
     values: List[str] = []
 
     def internal(node: Union[Tree, Token]) -> None:
-        if (
-            parameters
-            and isinstance(node, Tree)
-            and node.data == "identifier"
-            and node.children[0].data == "quoted_identifier"
-        ):
+        if parameters and isinstance(node, Tree) and node.data == "identifier":
             # This is the case where they reference a variable using
-            # the `#"Name of variable"` syntax.
+            # the `#"Name of variable"` or `Variable` syntax. It can be
```

Comment:
What was the previous assert doing here? No purpose?

Reply:
this broadens it from quoted_identifier -> identifier (which includes regular_identifier)